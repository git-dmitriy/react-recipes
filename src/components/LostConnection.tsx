import { IoCloudOfflineSharp } from 'react-icons/io5';

export const LostConnection = () => {
  return (
    <>
      <div className='flex flex-col items-center justify-center text-2xl'>
        <div>
          <IoCloudOfflineSharp className='text-6xl' />{' '}
        </div>
        <p>
          <strong>The Internet connection is lost</strong>
        </p>
      </div>
    </>
  );
};
