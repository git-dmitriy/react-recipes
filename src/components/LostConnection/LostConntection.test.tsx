import renderer from 'react-test-renderer';
import {LostConnection} from './LostConnection';
import {it, expect} from 'vitest';

it('should render correctly', () => {
    const tree = renderer.create(<LostConnection/>).toJSON();

    expect(tree).toMatchSnapshot();
});
