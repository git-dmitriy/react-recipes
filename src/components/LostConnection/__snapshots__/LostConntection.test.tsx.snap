// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render correctly 1`] = `
<div
  className="flex flex-col items-center justify-center text-2xl"
>
  <div>
    <svg
      className="text-6xl"
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        {
          "color": undefined,
        }
      }
      viewBox="0 0 512 512"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="m41.37 64 22.628-22.628L470.627 448l-22.628 22.627zm-2.65 148.78C13.39 235.88 0 267.42 0 304c0 36 14.38 68.88 40.49 92.59C65.64 419.43 99.56 432 136 432h228.12L110.51 178.39c-28.01 5.39-53.09 17.33-71.79 34.39zm437.87 194.45C499.76 388.78 512 361.39 512 328c0-61.85-48.44-95.34-97.75-102.64-6.52-41.18-24.05-76.4-51.11-102.46A153.57 153.57 0 0 0 256 80c-30.47 0-58.9 8.62-83.07 25l302.82 302.86c.25-.21.57-.41.84-.63z"
      />
    </svg>
     
  </div>
  <p>
    <strong>
      The Internet connection is lost
    </strong>
  </p>
</div>
`;
